model/serverSelect.js

```diff
@@ -1,16 +1,36 @@
 module.exports = function(app, io){
 
+	//Testing values
```

Comment:
yea i used 'model' because of lack of better word that I could think of at the time, but as i wrote the code alot of it was vaguely similarly related but still different as its http and socket, will probably separate and organise when I do a lot more socket.io work